docs/pages/installation.mdx

```diff
@@ -167,12 +154,14 @@ $ curl (=teleport.teleport_install_script_url=) | bash -s ${TELEPORT_VERSION?} $
 ### Package repositories
 
 The [one-line installation script](#one-line-installation-script) automatically
-detects a package manager on the local system and uses it to install Teleport
-from an upstream repository.
-
-If your system does not support the one-line installation script, read the
-instructions in this section for instructions on working with Teleport package
-repositories.
+installs Teleport from the official Teleport CDN.
+
+If your system does not support the one-line installation script or the
+`teleport-update` command, read the instructions in this section for
+instructions on working with Teleport package repositories directly.
+Note that package installations have similar requirements to `teleport-update`.
+The package-based installation workflow is deprecated for Teleport agents,
```

Comment:
Does this mean that the one-line installation script is deprecated, or just manually choosing a package manager and installing a package from a repo?